File: sort-algorithms/bubble.py
def bubbleSort(lst1):
    for i in range(1, len(lst1), 1):
        for j in range(len(lst1) - i):
            if lst1[j] > lst1[j+1]:
                lst1[j], lst1[j+1] = lst1[j+1], lst1[j]
    print(lst1)

    return lst1

bubbleSort([4, 3, 5, 2, 1])

File: sort-algorithms/bubble.js
function bubbleSort(ar) {
    for(let i = 0; i <= ar.length - 1; i++) {
        for(let j = 0; j < ar.length - 1; j++) {
            if(ar[j] > ar[j + 1]) {
                let temp = ar[j];
                ar[j] = ar[j + 1];
                ar[j + 1] = temp;
            }
            // console.log(ar);
        }
        // console.log(ar);
    }
    // console.log(ar);
}

bubbleSort([4, 3, 6, 7, 0]);
